README.md

```diff
@@ -1,3 +1,8 @@
+# :warning: End-of-Life Notice :warning:
+`symphony-api-client-node` will be supported until **end of 2021**, please consider moving [BDK for Java](https://github.com/finos/symphony-bdk-java) 
```

Comment:
_moving to_ ?

I'm not sure we can really offer that as an alternative :)
maybe we can just mention that those are the ones we support and just link to the developer's REST API doc?